src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -62,10 +67,19 @@ public void setUp() {
         handler = new FetchCristinProjects(mockCristinApiClient, environment);
     }
 
+    @ParameterizedTest
+    @ArgumentsSource(TestPairProvider.class)
+    void handlerReturnsExpectedBodyWhenRequestInputIsValid(String queryResponse,
```

Comment:
General comment: If this test ever breaks, it could be hard to find out why. But I see that there can be no quick solutions to this problem now.

Reply:
Yes. The test needs to be in place for next PR